deployment.rb

```diff
@@ -91,6 +91,7 @@ def load_configuration()
     'dynamo_tables_table'         => "#{rack_env}_tables",
     'dynamo_tables_index'         => "channel_id-table_name-index",
     'dynamo_properties_table'     => "#{rack_env}_properties",
+    'throttle_data_apis'          => rack_env == :production,
```

Comment:
Agree we should not limit to production.

Redis is used in Netsim and also available in the staging and test environments (otherwise we wouldn't be able to test Netsim.)   See net_sim_api in shared.  

We might want to explicitly define geocode_redis_url to the localhost address in those environments as well to eliminate any confusion.